rep-2002.rst

```diff
@@ -32,14 +32,30 @@ During the pre-release phase for stable ROS distributions releases in the rollin
 Release process
 ===============
 
+Automated release migration tool
+--------------------------------
+
+To make a new rosdistro, target, from an existing rosdistro, source. An automated migration tool is available to to avoid manual labor.
+
+A snaphot of the source distro will be captured, most likely a git commit-ish.
+The release manager should then setup the target rosdistro with all the appropriate configurations, such as target platforms and architectures.
+The tool will then iterate through all repositories in the snapshot of the source rosdistro and attempt to automatically run bloom release on the package at the specific version in the snapshot.
+If it fails the maintainer will be needed to resolve any issues.
+The tool will continue skipping that package and all downstream repositories.
```

Comment:
If this is meant to describe the migration script, we don't transitively skip. In fact we avoid needing to topologically sort altogether by allowing bloom to assume everything will be in the release and then removing what doesn't make it. We'll still try to bloom everything but the revoked release of a dependency will prevent the downstream jobs from being created on the buildfarm.

Reply:
ok, I just pulled that form the description